src/sentry/eventstore/base.py

```diff
@@ -6,76 +6,107 @@
 from sentry import nodestore
 from sentry.utils.services import Service
 
-Column = namedtuple("Column", "event_name transaction_name alias")
+Column = namedtuple("Column", "event_name transaction_name discover_name alias")
```

Comment:
Did you check none is accessing this tuple positionally ? Otherwise all those places would return the wrong result

Reply:
Yeah. Glad we changed this to namedtuple earlier.